gap/weights.gi

```diff
@@ -780,3 +780,50 @@ function(digraph, start, destination)
 
   return flows;
 end);
+
+#############################################################################
+# 6. Random edge weighted digraphs
+#############################################################################
+
+BindGlobal("DIGRAPHS_RandomEdgeWeightedDigraphFilt",
```

Comment:
I still think these two functions aren't necessary, and you're really just installing two methods in a more complicated way. Please look at `RandomDigraph` and implement this in the same way.